sass/components/_categories.scss

```diff
@@ -0,0 +1,140 @@
+@charset 'UTF-8';
```

Comment:
Similar to the other PR, this CSS should live in the same folders as the accompanying components